features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/PrivacyOverrideExtensions.kt

```diff
@@ -22,3 +22,31 @@ fun View.datadogSessionReplayHidden(hide: Boolean) {
         this.setTag(R.id.datadog_hidden, null)
     }
 }
+
+/**
+ * Allows overriding the image privacy for a view in Session Replay.
+ *
+ * @param privacy the new privacy level to use for the view
+ * or null to remove the override.
+ */
+fun View.datadogSessionReplayImagePrivacy(privacy: ImagePrivacy?) {
```

Comment:
opinionated:
* The API names seems too long to use, I think we can use some abbreviation for "datadog" or "SessionReplay" since user can easily get the context by clicking into the function.
* Although it is declared in doc, It is still not intuitive for me that passing a null value means removing the override, probably a dedicated api to clear the override?
* What is the consequence if user call this API on a `TextView`, should we specify the case in the doc?

Reply:
I think that for the naming issues in general we should have a discussion (also on whether to have a separate api for removing the overrides). 

Regarding the textview, there could be compound drawables. Also the overrides are propagated downwards by nearest parent so even for cases where there's ostensibly no effect on the direct view, there may be views further down the tree that it would change the masking for. We can cover this in the docs.